nvm.sh

```diff
@@ -795,12 +790,8 @@ nvm_ls_current() {
     nvm_add_iojs_prefix "$(iojs --version 2>/dev/null)"
   elif nvm_tree_contains_path "${NVM_DIR}" "${NVM_LS_CURRENT_NODE_PATH}"; then
     local VERSION
-    VERSION="$(node --version 2>/dev/null)"
-    if [ "${VERSION}" = "v0.6.21-pre" ]; then
-      nvm_echo 'v0.6.21'
-    else
-      nvm_echo "${VERSION}"
-    fi
+    VERSION=${${NVM_LS_CURRENT_NODE_PATH#$NVM_DIR/versions/node/}%/bin/node}
```

Comment:
This is reporting "bad substitution" - https://travis-ci.org/creationix/nvm/jobs/320732790#L1029